modules/plugins/languages/tex/build/builders/tectonic.nix

```
@@ -0,0 +1,203 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+} @ moduleInheritancePackage: let
+  # The name of the builder
+  name = "tectonic";
+
+  # The builder template
+  template = import ./builderTemplate.nix;
+
+  inherit (lib.options) mkOption mkEnableOption;
+  inherit (lib.types) enum ints listOf package str;
+  inherit (lib.nvim.config) mkBool;
+  inherit (builtins) concatLists elem map toString;
+
+  cfg = config.vim.languages.tex;
+in (
+  template {
+    inherit name moduleInheritancePackage;
+
+    options = {
+      enable = mkEnableOption "Whether to enable Tex Compilation Via Tectonic";
```

```suggestion
      enable = mkEnableOption "Tex Compilation Via Tectonic";
```
`mkEnableOtion` prepends "Whether to enable..." to the description already.